Makefile

```
@@ -107,7 +107,7 @@ slow_tests_diffusers: test_installs
 
 # Run text-generation non-regression tests
 slow_tests_text_generation_example: test_installs
-	python -m pip install triton==3.1.0 autoawq
+	python -m pip install triton==3.1.0 autoawq --no-deps
```

```suggestion
	python -m pip install -r examples/text-generation/requirements_awq.txt
```
Let's use the requirements file everywhere so that we don't have to modify the requirements in 3 different places